Dockerfile

```diff
@@ -95,7 +97,7 @@ COPY scripts/obodash /tools
 RUN chmod +x /tools/obodash && \
     git clone --depth 1 https://github.com/OBOFoundry/OBO-Dashboard.git && \
     cd OBO-Dashboard && \
-    python3 -m pip install -r requirements.txt && \
+    python3 -m pip install -r requirements.txt --break-system-packages && \
```

Comment:
> The OBO Dashboard must now be installed with --break-system-packages, otherwise pip will flatly refuse to install it system-wide.

I wonder if this is very suboptimal and we should, instead, install OBO dashboard as a pypi package and install it the usual way? Why is `--break-system-packages` not needed in `docker/builder/Dockerfile`?

Reply:
What are the break system packages? I can update to the latest version, if possible.

Reply:
> I wonder if this is very suboptimal and we should, instead, install OBO dashboard as a pypi package and install it the usual way?

We could, but then we would need actual _releases_ of the OBO Dashboard package whenever we want to update it in the ODK. For now, the OBO Dashboard is effectively working in a “rolling release” fashion: [it doesn’t do releases](https://github.com/OBOFoundry/OBO-Dashboard/releases) and we always install it from the tip of the master branch. That would need to change.

Reply:
> That would need to change.

I can create a release right now.

Reply:
> Why is `--break-system-packages` not needed in `docker/builder/Dockerfile`?

Because in `docker/builder/Dockerfile` we are _not_ installing Python packages to the system-wide directory: we are installing them to a staging area, from where we copy them to the ODKLite/ODKFull images.